crates/bevy_pbr/src/pbr_material.rs

```diff
@@ -12,10 +12,11 @@ use bevy_render::{
 /// <https://google.github.io/filament/Material%20Properties.pdf>.
 ///
 /// May be created directly from a [`Color`] or an [`Image`].
-#[derive(AsBindGroup, Debug, Clone, TypeUuid)]
+#[derive(AsBindGroup, Reflect, FromReflect, Debug, Clone, TypeUuid)]
 #[uuid = "7494888b-c082-457b-aacf-517228cc0c22"]
 #[bind_group_data(StandardMaterialKey)]
 #[uniform(0, StandardMaterialUniform)]
+#[reflect(Default)]
```

Comment:
Do we want other reflect traits here too?

Reply:
We do have it for `Debug` (not as `ReflectDebug` but as a "special trait")